clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -38,11 +39,30 @@ import (
 const (
 	schedulerLatencyMetricName = "SchedulingMetrics"
 
-	e2eSchedulingDurationMetricName       = model.LabelValue(schedulermetric.SchedulerSubsystem + "_e2e_scheduling_duration_seconds_bucket")
-	schedulingAlgorithmDurationMetricName = model.LabelValue(schedulermetric.SchedulerSubsystem + "_scheduling_algorithm_duration_seconds_bucket")
+	e2eSchedulingDurationMetricName           = model.LabelValue(schedulermetric.SchedulerSubsystem + "_e2e_scheduling_duration_seconds_bucket")
+	schedulingAlgorithmDurationMetricName     = model.LabelValue(schedulermetric.SchedulerSubsystem + "_scheduling_algorithm_duration_seconds_bucket")
+	frameworkExtensionPointDurationMetricName = model.LabelValue(schedulermetric.SchedulerSubsystem + "_framework_extension_point_duration_seconds_bucket")
+	schedulingLatencyMetricName               = model.LabelValue(schedulermetric.SchedulerSubsystem + "_" + schedulermetric.DeprecatedSchedulingDurationName)
 
-	schedulingLatencyMetricName = model.LabelValue(schedulermetric.SchedulerSubsystem + "_" + schedulermetric.DeprecatedSchedulingDurationName)
-	singleRestCallTimeout       = 5 * time.Minute
+	singleRestCallTimeout = 5 * time.Minute
+)
+
+var (
+	extentionsPoints = []string{
+		"PreFilter",
+		"PreFilterExtensionAddPod",
+		"PreFilterExtensionRemovePod",
```

Comment:
please remove those, they are not really an extension point, the metrics recorded for those are per plugin.

Reply:
note that those are included in the preemption latency metric.